React/game/src/app/screen/Game/index.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+
+import Board from './Board';
+import styles from './styles.scss';
+import { serialWin } from './constants';
+
+class Game extends Component {
+  state = {
+    moves: 0,
+    gameEnd: false,
+    turn: 'X',
+    turnWinner: '',
+    history: [
+      {
+        board: Array(9).fill(''),
+        id: 0
+      }
+    ]
+  };
+
+  playerOne = 'X';
+  playerTwo = 'O';
```

Comment:
move these `playerOne` and `playerTwo` to the top of the class

Reply:
👍